ament_index_cpp/src/get_resources.cpp

```diff
@@ -66,8 +74,11 @@ get_resources(const std::string & resource_type)
     do {
       // ignore directories
       if ((find_data.dwFileAttributes & FILE_ATTRIBUTE_DIRECTORY) == 0) {
-        if (resources.find(find_data.cFileName) == resources.end()) {
-          resources[find_data.cFileName] = base_path;
+        // ignore files starting with a dot
+        if (find_data.cFileName[0] != '.') {
+          if (resources.find(find_data.cFileName) == resources.end()) {
+            resources[find_data.cFileName] = base_path;
+          }
```

Comment:
Since the nesting level gets deeper here I would suggest to invert the logic and `continue` in order to keep the nesting shallow:

```
// ignore directories
if (find_data.dwFileAttributes & FILE_ATTRIBUTE_DIRECTORY) {
  continue;
}
// ignore files starting with a dot
if (find_data.cFileName[0] == '.') {
  continue;
}
if (resources.find(find_data.cFileName) == resources.end()) {
  resources[find_data.cFileName] = base_path;
}
```